domains/abhilash.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal portfolio",
+    "repo": "https://github.com/ostrichization/ostrichization.github.io",
+    "owner": {
+        "username": "<github-username>",
```

Comment:
You should put your gtihub username here

Reply:
Sorry! Fixed!